code/game/objects/items/devices/defib.dm

```diff
@@ -262,10 +262,12 @@
 
 //Checks for various conditions to see if the mob is revivable
 /obj/item/weapon/shockpaddles/proc/can_defib(mob/living/carbon/human/H) //This is checked before doing the defib operation
-	if((H.species.flags & NO_SCAN))
+	if((H.species.flags & NO_SCAN && !(H.species.reagent_tag & IS_VOX)))
```

Comment:
You kinda cocked up the change request, dude. Change requests are mutually exclusive regarding the same file.